ecs-agent/tmds/utils/netconfig/netconfig_linux.go

```diff
@@ -69,3 +72,28 @@ func DefaultNetInterfaceName(netlinkClient netlinkwrapper.NetLink) (string, erro
 	}
 	return "", nil
 }
+
+func GetInterfaceIPAddresses(nw netwrapper.Net, ifaceName string) ([]string, error) {
```

Comment:
Some documentation would be nice.

Reply:
How about renaming this so it's clear that only global addresses would be returned?

Reply:
Yes, let me update this.

Reply:
Updated in a new rev.